### PR TITLE
Feature measure_concentration

### PR DESCRIPTION
**Pull request by bmiles**
This PR adds support for the `measure_concentration` operation.

### CODE REVIEW

File: autoprotocol/protocol.py [hunk 1]
```diff
@@ -4348,3 +4348,63 @@ def _ref_containers_and_wells(self, params):
                 parameters[str(k)] = v
 
         return parameters
+
+    def measure_concentration(self, wells, volume, dataref, measurement):
```
Comment (transcripticpll) [inline autoprotocol/protocol.py]: set volume="2:microliter" as default for ease of use

File: test/protocol_test.py [hunk 2]
```diff
@@ -1121,3 +1121,32 @@ def test_autopick(self):
...
+    def test_measure_concentration_multi_sample_class(self):
+        p = Protocol()
+        test_plate = p.ref("test_plate", id=None, cont_type="96-flat", storage=None, discard=True)
+        for well in test_plate.all_wells():
+            well.set_volume("150:microliter")
+        for i, sample_class  in enumerate(["ssDNA", "DNA", "RNA", "protein"]):
+            p.measure_concentration(test_plate.well(i), Unit(2, "microliter"), dataref="mc_test_%s" % sample_class, measurement=sample_class)
+        self.assertEqual(len(p.instructions), 4)
+        for i, type in enumerate(["ssDNA", "DNA", "RNA", "protein"]):
+            self.assertEqual(p.as_dict()["instructions"][i]["measurement"], type)
```
Comment (transcripticpll) [inline test/protocol_test.py]: nest this before 1150 so the measurement list is always up to date

**Comment by bmiles** [comment]
@transcripticpll hope that addresses the comments

**Comment by transcripticpll** [comment]
@bmiles looks good. thanks

**Comment by transcripticpll** [comment]
Oh, sorry, really quick, can you update the changelog?

**Comment by transcripticpll** [comment]
Oh, sorry, really quick, can you update the changelog?

**Comment by bmiles** [comment]
Done

**Pull request merged**